src/tools/Reporting/Reporting/Reporter.cs

```diff
@@ -146,10 +146,7 @@ private string LeftJustify(string str, int width)
         {
             return String.Format("{0,-" + width + "}", str);
         }
-        
-        private bool CheckEnvironment()
```

Comment:
By any chance would we want to check more than one environment variables someday, so we need to keep it as a method here?

Reply:
Or your intention is to make ```CheckEnvironment``` more explicit by just having individual ```InLab``` boolean, so we know what it's doing.

Reply:
Yeah, I would say that the original name was not well thought out. It was really just intended to asses whether or not we were in the lab. All the other interesting work happens in `Init`.